eng/pipelines/templates/steps/download-package-artifacts.yml

```diff
@@ -0,0 +1,18 @@
+parameters:
+  TargetPath: $(Build.ArtifactStagingDirectory)
+
+steps:
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        artifactName: 'packages_windows'
```

Comment:
Do we need OS specific artifacts for rust? If not then you shouldn't need this like python does.